test/run/ch-fromhost.bats

```diff
@@ -109,30 +109,33 @@ fromhost_ls () {
     fromhost_clean "$img"
 }
 
-@test 'ch-fromhost (CentOS)' {
+@test 'ch-fromhost (Debian)' {
     scope full
-    prerequisites_ok centos7
-    img=${ch_imgdir}/centos7
+    prerequisites_ok debian9
+    img=${ch_imgdir}/debian9
+
+    libpath=$(ch-fromhost --lib-path "$img")
+    echo "libpath: ${libpath}"
```

Comment:
I think I would leave this hardcoded in both actually, to make our expectations clear. If that path shuffles around, we should know.

Reply:
I'll change to hard coded paths.